modules/deploy/pages/deployment-option/cloud/managed-connectors/sizing-connectors.adoc

```diff
@@ -14,6 +14,13 @@ Before setting up connector tasks, consider the following:
 
 - For sink connectors, parallelism is achieved by evenly distributing configured topic partitions for the connector amongst connector tasks. The number of partitions must be equal to or greater than the number of tasks.
 
+[NOTE]
+====
+Connectors tasks limit depends on the cluster tier.
+
+Contact https://support.redpanda.com/hc/en-us/requests/new[Redpanda support^] if you need to configure more than 500 tasks per cluster. 
```

Comment:
```suggestion
The maximum number of connector tasks varies based on the xref:deploy:deployment-option/cloud/byoc-tiers.adoc[ cluster tier]. To configure more than 500 tasks for a cluster, contact https://support.redpanda.com/hc/en-us/requests/new[Redpanda support^]. 
```
@bochenekmartin Is this suggestion correct? We don't document the maximum number of connector tasks for each tier. (We list maximum connections, but not maximum connector tasks.) Should we include this link, or does it make it more confusing?

Reply:
I think the link may be confusing, because it's BYOC specific. 
But anyway, I'll rollback this change, to avoid confusions.